Settings.h

```diff
@@ -36,6 +36,10 @@
 
 // Define bluetooth scan parameters
 #define scanInterval 2 // Define the interval in seconds between scans
+#define singleScanTime 10 // Define the duration of a single scan in seconds
+#define activeScan true // Active scan uses more power, but get results faster
+#define bleScanInterval 0x80 // (milliseconds) The time interval from when the Controller started its last LE scan until it begins the subsequent LE scan
```

Comment:
This is, in fact, not what this setting implies. Instead, it relates to how the ESP32 will determine sharing of the single on-board antenna between the WiFi and BLE operations. While I appreciate breaking this out into its own setting, I will add a different comment about the need for modification, with a strong recommendation that it is not modified from its initial setting.

Mis-configuration of these two parameters will result in repeated disconnects of the device as it will fail to maintain a connection to the MQTT server, and/or manifest as a core panic.

A similar concept applies to the `bleScanWindow` setting as well.

 See https://www.espressif.com/sites/default/files/documentation/ESP32_FAQs__EN.pdf  section 5.3.1 for more detailed info.